requirements/dev.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    pip-compile --output-file=requirements/dev.txt requirements/dev.in
 #
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos  # via -r requirements/dev.in
+-e git+https://github.com/kytos-ng/kytos.git@feature/mongo_client#egg=kytos  # via -r requirements/dev.in
```

Comment:
I've had to temporarily pin kytos core branch on requirements/dev.txt on this commit d98c3aa7c4349e070661633ed88db2e3e5774d1e, I'll revert this commit once [kytos core PR 202](https://github.com/kytos-ng/kytos/pull/202) lands.

Reply:
Since kytos core PR 202 has landed, I've reverted this commit